Sources/Fuzzilli/Core/ProgramBuilder.swift

```diff
@@ -333,32 +332,22 @@ public class ProgramBuilder {
     ///
     private var varMaps = [VariableMap<Variable>]()
     /// Formatted ProgramTypes structure for easier adopting of runtimeTypes
-    private var typeMaps = [[[(Variable, Type)]]]()
+    private var runtimeTypesMaps = [[[(Variable, Type)]]]()
     
     /// Prepare for adoption of variables from the given program.
     ///
     /// This sets up a mapping for variables from the given program to the
     /// currently constructed one to avoid collision of variable names.
     public func beginAdoption(from program: Program) {
         varMaps.append(VariableMap())
-        typeMaps.append(formatTypeMap(from: program))
-    }
-
-    // Format ProgramTypes struct so searching for type changes at instruction is easier
-    private func formatTypeMap(from program: Program) -> [[(Variable, Type)]] {
-        var typeMap: [[(Variable, Type)]] = Array(repeating: [], count: program.size)
-        for (variable, typeData) in program.runtimeTypes {
-            for typeIndexPair in typeData {
-                typeMap[typeIndexPair.index].append((variable, typeIndexPair.type))
-            }
-        }
-        return typeMap
+        // Append inverted ProgramTypes for easier search for type changes at instruction
```

Comment:
Nit: this comment is outdated, but I think it can be removed entirely now with the new method names